internal/impl/io/input_websocket.go

```diff
@@ -181,6 +194,22 @@ func (w *websocketReader) Connect(ctx context.Context) error {
 		}
 	}
 
+	if len(w.openMsgs) > 0 {
+		if openMsgType == websocket.TextMessage {
+			for _, msg := range w.openMsgs {
+				if len(msg) == 0 {
+					continue
+				}
+				// TODO: will need remove such log later cause it may contain sensitive info
+				w.log.Infof("sending open message %s to server", string(msg))
```

Comment:
I still think there is no need to print this out if you have already verified Orbit using yaml. The interface between Neutron and Orbits are pipeline yaml. When debug, you should be able to dump the yaml generated by Neutron and run it directly on Orbit. It should be fairly easy to figure out where causes the issue in case of any.

By adding this log and removing it later, you will need to the following things additionally
1. Create a one line PR to remove this log
2. Create a new orbit release
3. Update Atom to deploy the new orbit version

This seems to be even more time-consuming